samples/cpp/benchmark_app/main.cpp

```diff
@@ -175,15 +248,22 @@ int main(int argc, char* argv[]) {
             load_config(FLAGS_load_config, config);
         }
 #endif
-        /** This vector stores paths to the processed images **/
-        std::vector<std::string> inputFiles;
-        parseInputFilesArguments(inputFiles);
+        /** This vector stores paths to the processed images with input names**/
+        auto inputFiles = parseInputArguments();
 
         // ----------------- 2. Loading the Inference Engine
         // -----------------------------------------------------------
         next_step();
 
         Core ie;
+        if (FLAGS_d == "TEMPLATE") {
+#ifdef _DEBUG
+            ie.RegisterPlugin("templatePlugind", "TEMPLATE");
+#else
+            ie.RegisterPlugin("templatePlugin", "TEMPLATE");
+#endif
```

Comment:
why do we need it? this plugin is not for production.

Reply:
Ok, removed https://github.com/openvinotoolkit/openvino/pull/7788/commits/d255d1f0e6806f0a0fe11fe1bbe689e9b1e58828